### ISSUE TITLE
Freiwilligen-Ansicht

### ISSUE THREAD

**Issue by rebeccakurz** [barbalex/apf2#620]
Hoi Alex

Eine Freiwillige hat nachgefragt, ob wir etwas an der Ansicht der Freiwilligen-Kontrollen geändert hätten, sie sehe ihre Kontrollen nicht mehr. 
Ich habe mich nun mit dem Freiwilligen-Test-Account "Test Tester" angemeldet und werde tatsächlich anstatt zu der Freiwilligen-Ansicht auf die "normale" FloraDB geleitet. Kann das sein, dass das bei allen Freiwilligen der Fall ist?
Könntest du dir das mal anschauen?
![Bildschirmfoto 2023-03-20 um 08 19 03](https://user-images.githubusercontent.com/30288503/226272055-1eba2c34-5721-4711-a07e-dca126d7b307.png)

**Comment by barbalex** [comment]
Komisch ist, dass es in der live Version den Test Tester weder als Benutzer noch als Adresse mehr gibt. Hast du ihn gelöscht?

**Comment by rebeccakurz** [comment]
Nein, ich bin gerade noch als Test Tester angemeldet, vielleicht darum? 
![Bildschirmfoto 2023-03-20 um 11 03 15](https://user-images.githubusercontent.com/30288503/226307770-80340774-e474-45fe-b8f8-900d59fb790a.png)
Ich melde mich mal ab...

**Comment by rebeccakurz** [comment]
![Bildschirmfoto 2023-03-20 um 11 04 34](https://user-images.githubusercontent.com/30288503/226308018-695d2d9c-a8ff-48a8-bede-5e62c8d4c118.png)
Also bei mir ist er noch drin...

**Comment by barbalex** [comment]
Aha, die Sortierung sortiert Grossbuchstaben zuerst 🙄

**Comment by barbalex** [comment]
Ich brauch etwas länger,, um diese Problem zu lösen. Mache am Nachmittag weiter

**Comment by barbalex** [comment]
Ich hatte in meiner lokalen Installation das Problem, dass die rolle `apflora-freiwillig` nicht die nötigen Rechte hatte. Ist mir völlig unerklärlich, wie das passieren konnte und hat mich 2 Stunden gekostet.

Auf dem Server ist das nicht der Fall.

Jetzt kann ich endlich dem eigentlichen Problem nachgehen.

**Comment by barbalex** [comment]
O.k., sollte in Version 1.82.0 behoben sein.

Hab dann noch gesehen, dass Freiwillige nicht abmelden können. Habe diese Möglichkeit ergänzt.

**Comment by rebeccakurz** [comment]
Das sieht schon besser aus, aber vorher wurden den Freiwilligen gleich nach der Anmeldung ihre Formulare angezeigt, so dass sie diese nur noch ausfüllen mussten (also dass sie gar keinen Strukturbaum etc. sehen, sondern nur die Eingabemaske der Freiwilligen-Formulare, so wie auf dem Screenshot unten).
![Bildschirmfoto 2023-03-20 um 15 46 48](https://user-images.githubusercontent.com/30288503/226376686-d8cfe803-4d2a-4b23-ba16-5d272429946f.png)

**Comment by rebeccakurz** [comment]
Okay, super, vielen Dank!

**Comment by barbalex** [comment]
Wenn keine Kontrollen existieren sollte so was hier kommen:
<img width="614" alt="2023-03-20_16h10_45" src="https://user-images.githubusercontent.com/669783/226383392-be87cb1d-5c87-416d-9566-e57dddd3edc1.png">

(auf meinem Entwicklungs-PC gibt es für Test Tester noch keine)

**Comment by rebeccakurz** [comment]
Danke!
